src/routes/api/+page.ts

```
@@ -2,13 +2,15 @@ import { fetch_zipped, mp_build_bucket } from '$lib/api'
 import type { SummaryDoc } from '$lib/material'
 
 export const ssr = false
-const mp_build_url = `https://materialsproject-build.s3.amazonaws.com?delimiter=%2F&prefix=collections%2F2022-10-28%2F`
+const mp_build_url =
+  `https://materialsproject-build.s3.amazonaws.com?delimiter=%2F&prefix=collections%2F2022-10-28%2F`
 
 async function fetch_bucket_names() {
   const text = await fetch(mp_build_url).then((res) => res.text())
 
   // mock DOMParser when server-rendering
   if (typeof globalThis.DOMParser === `undefined`) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)